docs/blog/pydantic-2-alpha.md

```diff
@@ -0,0 +1,40 @@
+# Pydantic 2 Alpha Release
+
+<aside class="blog" markdown>
+![Samuel Colvin](/img/samuelcolvin.jpg)
+<div markdown>
+  **Samuel Colvin** &bull;&nbsp;
+  [:material-github:](https://github.com/samuelcolvin) &bull;&nbsp;
+  [:material-twitter:](https://twitter.com/samuel_colvin) &bull;&nbsp;
+  :octicons-calendar-24: April 01, 2023 &bull;&nbsp;
+  :octicons-clock-24: 5 min read
+</div>
+</aside>
+
+---
+
+We're excited to announce the first alpha release of Pydantic 2!
+
+This first Pydantic 2 alpha no April Fool's joke. After months of work, we invite you to explore the improvements we've made and give us your feedback. We look forward to hearing your thoughts and working together to improve the library.
```

Comment:
```suggestion
This first Pydantic 2 alpha is no April Fool's joke. After months of work, we invite you to explore the improvements we've made and give us your feedback. We look forward to hearing your thoughts and working together to improve the library.
```
Beyond the typo fix above, I feel like I want to emphasize the value of user feedback in some way. Something like:
> Your feedback will be a critical part of ensuring that we have made the right tradeoffs with the API changes in v2.

That's a little wordy though, and not quite right anyway. I'm wanting to convey the points:
* "please let us know if you don't like the changes so we can make sure pydantic remains a library you _enjoy_ using", and 
* "please let us know if this breaks your usage of pydantic so we can fix it, or at least describe a migration path"

Don't have to add this, and I see you addressed the second bullet below, but maybe something here will resonate with you and you'll be able to wordsmith it a bit better than I could. Feel free to ignore if there's not a good way to make this point, though.

Reply:
I think it's very much worth spelling this out as explicitly as you have here